packages/core/src/components/tooltip/tooltip.tsx

```diff
@@ -31,7 +31,7 @@ const applyMaxSize = {
   styleUrl: "tooltip.scss",
   shadow: true,
 })
-export class Tooltip {
+export class Tooltip implements ComponentInterface {
```

Comment:
Tijdens debuggen werd ik gek van WebStorm die terecht stelt dat er ongebruikte class methods zijn. Ik heb deze interface toegevoegd (wat uberhaupt de conventie is) om die meldingen weg te zetten.